interface/main/calendar/find_patient_popup.php

```diff
@@ -46,239 +46,254 @@
- </tr>
-</thead>
-<tbody id="searchResults">
-    <?php
-    if (is_countable($result)) {
-        foreach ($result as $iter) {
-            $iterpid   = $iter['pid'];
-            $iterlname = $iter['lname'];
-            $iterfname = $iter['fname'];
-            $itermname = $iter['mname'];
-            $iterdob   = $iter['DOB'];
-
-            // If billing note exists, then it gets special coloring and an extra line of output
-            // in the 'name' column.
-            $trClass = "oneresult";
-            if (!empty($iter['billing_note'])) {
-                $trClass .= " billing";
-            }
+    <div class="table-responsive-sm">
+        <div id="searchCriteria" class="bg-light p-2 pt-3">
+            <form method='post' name='theform' id="theform" action='find_patient_popup.php?<?php if (isset($_GET['pflag'])) {
+                echo "pflag=0";
+                                                                                           } ?>'>
+                <div class="form-row">
+                    <label for="searchby" class="col-form-label col-form-label-sm col"><?php echo text(xl('Search by:')); ?></label>
+                    <select name='searchby' id='searchby' class="form-control form-control-sm col">
+                        <option value="Last"><?php echo text(xl('Name')); ?></option>
+                        <!-- (CHEMED) Search by phone number -->
+                        <option value="Phone"<?php if (!empty($searchby) && ($searchby == 'Phone')) {
+                            echo ' selected';
+                                             } ?>><?php echo xlt('Phone'); ?></option>
+                        <option value="ID"<?php if (!empty($searchby) && ($searchby == 'ID')) {
+                            echo ' selected';
+                                          } ?>><?php echo xlt('ID'); ?></option>
+                        <option value="SSN"<?php if (!empty($searchby) && ($searchby == 'SSN')) {
+                            echo ' selected';
+                                           } ?>><?php echo xlt('SSN'); ?></option>
+                        <option value="DOB"<?php if (!empty($searchby) && ($searchby == 'DOB')) {
+                            echo ' selected';
+                                           } ?>><?php echo xlt('DOB'); ?></option>
+                    </select>
+                    <label for="searchparm" class="col-form-label col-form-label-sm col"><?php echo text(xl('for:')); ?></label>
+                    <input type='text' class="form-control form-control-sm col" id='searchparm' name='searchparm' size='12' value='<?php echo attr($_REQUEST['searchparm'] ?? ''); ?>' title='<?php echo text(xl('If name, any part of lastname or lastname,firstname'), ENT_QUOTES); ?>' />
+                    <div class="col">
+                        <input class='btn btn-primary btn-sm' type='submit' id="submitbtn" value='<?php echo text(xl('Search'), ENT_QUOTES); ?>' />
```

Comment:
Have a left over `ENT_QUOTES` above
and it should be attr() or could go to xla() if drop the xl()